.travis.yml

```diff
@@ -7,8 +7,5 @@ services:
 # a lightweight environment
 language: c
 
-install:
-    - ci/travis-install.sh
```

Comment:
This is still needed! This code is the one in charge to build an image specifically for Ocean (based on the `dlang` image).